lib/fastlane/plugin/wpmreleasetoolkit/actions/ios/ios_finalize_prechecks.rb

```diff
@@ -6,8 +6,10 @@ def self.run(params)
 
         require_relative '../../helper/ios/ios_version_helper'
         require_relative '../../helper/ios/ios_git_helper'
+        require_relative '../../helper/git_helper'
 
-        UI.user_error!('This is not a release branch. Abort.') unless other_action.git_branch.start_with?('release/')
+        current_branch = Fastlane::Actions.git_branch_name_using_HEAD
```

Comment:
Should `current_git_branch` be used here too?

Reply:
🤦  Yes it should be, thank you for catching that!

Addressed in f256ae5925ebe77a372d0fa330e4ef692a5c5d6b.